docs/guide/getstarted/configuration.md

```diff
@@ -106,6 +106,9 @@ Default: *90000*
 ### connectionRetryCount
 Count of request retries to the Selenium server
 
+###specFileRetries
```

Comment:
I don't think the type and default description below applies to the specFileRetries option also let's be consistent and keep a space between `###` and `specFileRetries`

Reply:
I think you may right I pinged @bennieswart because I am a little confused myself on this.